manifests/init.pp

```diff
@@ -221,6 +224,7 @@
   Boolean $enable_trusted_auth = $candlepin::params::enable_trusted_auth,
   Optional[String] $consumer_system_name_pattern = $candlepin::params::consumer_system_name_pattern,
   Integer[0, 65535] $ssl_port = $candlepin::params::ssl_port,
+  String $host = $candlepin::params::host,
```

Comment:
Stdlib offers a type that's either a FQDN or an IP (including `0.0.0.0` and `::`)
```suggestion
  Stdlib::Host $host = $candlepin::params::host,
```